lib/install.js

```diff
@@ -808,13 +835,35 @@ Installer.prototype.printInstalledForHuman = function (diffs, cb) {
     report += actions.join(', ') + ' and ' + lastAction
   }
   report += ' in ' + ((Date.now() - this.started) / 1000) + 's'
+  // If we're listing an added-packages-contributors count,
+  // add a note about `npx thanks`.
+  if (contributors.size) {
+    report += '\nRun `npx thanks` for info on how to support contributors.'
+  }
 
   output(report)
   return cb()
 
   function packages (num) {
     return num + ' package' + (num > 1 ? 's' : '')
   }
+
+  function from (num) {
+    return ' from ' + num + ' contributor' + (num > 1 ? 's' : '')
```

Comment:
A tiny suggestion:

In the message "installed N packages from M contributors", it's ambiguous whether "from" applies to "packages" or "installed". Read the latter way, it sounds like packages are being installed from multiple places (like multiple contributing registries or something?) which might be confusing for less npm-savvy folks, esp those unfamiliar with the recent Twitter conversations motivating this change.

Is there a way to rephrase? s/from/created by/?

Reply:
"by"?